staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -157,19 +142,24 @@ var _ ObjectTracker = &tracker{}
 // of objects for the fake clientset. Mostly useful for unit tests.
 func NewObjectTracker(registry *registered.APIRegistrationManager, scheme ObjectScheme, decoder runtime.Decoder) ObjectTracker {
 	return &tracker{
-		registry: registry,
-		scheme:   scheme,
-		decoder:  decoder,
-		objects:  make(map[schema.GroupVersionKind][]runtime.Object),
+		scheme:  scheme,
+		decoder: decoder,
+		objects: make(map[schema.GroupVersionResource][]runtime.Object),
 	}
 }
 
-func (t *tracker) List(gvk schema.GroupVersionKind, ns string) (runtime.Object, error) {
+func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionKind, ns string) (runtime.Object, error) {
 	// Heuristic for list kind: original kind + List suffix. Might
 	// not always be true but this tracker has a pretty limited
 	// understanding of the actual API model.
+	if gvr.Version == "" {
+
```

Comment:
Removed. I was hesitating whether to set internal gvr version to empty or to "__internal".